src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -153,83 +224,105 @@ THREE.WebGLShadowMap = function ( _renderer, _lights, _objects ) {
 			var shadowMatrix = light.shadowMatrix;
 			var shadowCamera = light.shadowCamera;
 
-			//
+			_lightPositionWorld.setFromMatrixPosition( light.matrixWorld );
+			shadowCamera.position.copy( _lightPositionWorld);
 
-			shadowCamera.position.setFromMatrixPosition( light.matrixWorld );
-			_matrixPosition.setFromMatrixPosition( light.target.matrixWorld );
-			shadowCamera.lookAt( _matrixPosition );
-			shadowCamera.updateMatrixWorld();
+			// render shadow map for each cube face (if omni-directional) or
+			// run a single pass if not
 
-			shadowCamera.matrixWorldInverse.getInverse( shadowCamera.matrixWorld );
+			for ( var face = 0; face < faceCount; face++ ){				
+		
+				if( isCube ){
 
-			//
+					_lookTarget.copy( shadowCamera.position );
+					_lookTarget.add( cubeDirections[face] );
+					shadowCamera.up.copy( cubeUps[face] );
```

Comment:
I'm not sure what you mean about this one. It seems like this file is formatted appropriately. I double checked using this tool: http://zz85.github.io/mrdoobapproves/